compiler/noirc_evaluator/src/ssa/plonky2_gen/mod.rs

```diff
@@ -603,10 +603,75 @@ impl Builder {
                         let (bit_size_b, target_b) = self.get_integer(rhs)?;
                         assert!(bit_size_a == bit_size_b);
 
-                        let div = add_div_mod(&mut self.asm_writer, target_a, target_b).0;
-                        let zero = self.asm_writer.zero();
-                        let target = self.asm_writer.is_equal(div, zero);
-                        Ok(P2Value::make_boolean(target))
+                        let bit_size = usize::try_from(match bit_size_a {
```

Comment:
Looks like `bit_size_a` is actually `type_of_a`. Perhaps the function was changed at some point but the use site was not updated.

Please, rename.

Reply:
Done: https://github.com/blocksense-network/noir/pull/60/commits/7e8fdb5aac2698d6c381d78f95e709cde00a50d5